services/bettersocial/user/BetterSocialBlockUserV2.js

```diff
@@ -88,13 +178,11 @@ const BetterSocialBlockUserV2 = async (token, selfUserId, targetUserId, source,
     };
   }
 
+  // Send queue to update score
   BetterSocialScoreBlockUser(selfUserId, targetUserId, postId);
   BetterSocialScoreBlockUser(selfUserId, targetAnonymousUserId?.user_id, '');
 
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `BetterSocialScoreBlockUser` function calls should handle the possibility of `targetAnonymousUserId` being null or undefined to prevent potential errors.

```diff
- BetterSocialScoreBlockUser(selfUserId, targetAnonymousUserId?.user_id, '');
+ if(targetAnonymousUserId){
+   BetterSocialScoreBlockUser(selfUserId, targetAnonymousUserId.user_id, '');
+ }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->